daemon/src/projection.rs

```diff
@@ -858,27 +879,9 @@ impl xtra::Actor for Actor {
     type Stop = ();
     async fn started(&mut self, ctx: &mut xtra::Context<Self>) {
         let this = ctx.address().expect("we just started");
-        let pool = self.db.clone();
-
-        self.tasks.add_fallible(
-            {
-                let this = this.clone();
-
-                async move {
-                    let mut conn = pool.acquire().await?;
-                    let vec = db::load_all_cfd_ids(&mut conn).await?;
-
-                    for id in vec {
-                        let _: Result<(), xtra::Disconnected> = this.send(CfdChanged(id)).await;
-                    }
-
-                    anyhow::Ok(())
-                }
-            },
-            |e| async move {
-                tracing::warn!("Failed to do initial rehydratation of CFDs: {e:#}");
-            },
-        );
+        this.send_async_safe(Initialize)
+            .await
+            .expect("we just started");
```

Comment:
Can we double check that this `expect` is on the `xtra::Disconnected`?

Reply:
just checked, this is the case: https://github.com/itchysats/itchysats/blob/master/xtras/src/send_async_safe.rs#L40